src/app/tabs/run-tab.js

```diff
@@ -193,6 +193,23 @@ function updateAccountBalances (container, self) {
   })
 }
 
+function updateDeployedContractBalances (container, self) {
```

Comment:
instead of calling `self._deps.udapp.getBalanceInEther` each time we want to render a contract instance, i think we should use `setInterval` to retrieve all the instances and update the balance.
cc @ryestew